src/main/java/core/basesyntax/Factorial.java

```diff
@@ -4,8 +4,14 @@
  * <p>Реализуйте метод getFactorial(int number) который вовзращает факториал числа.
  * Реализовать с помощью рекурсии.</p>
  */
+
 public class Factorial {
     public int getFactorial(int number) {
-        return 0;
+        int result = 1;
+        if (number == 1 || number == 0) {
+            return result;
+        }
+        result = number * getFactorial(number - 1);
```

Comment:
do we need to create local variable in this case?

Reply:
Actually no, thanks